### ISSUE TITLE
Ion default file

### ISSUE THREAD

**Issue by thirstyice** [thirstyice/cSPACE-tech-tracking#2]
Fix the default Ion showfile

- [ ] Make the presets Presets (currently cues)
- [ ] Make the subs affect only the channels that they should affect (currently subs include color information for other channels)
- [ ] Make the sub layout more useful for busking
- [ ] Make the channel numbers make sense
- [ ] Magic sheet
- [ ] Color pallets

**Comment by thirstyice** [comment]
- [ ] Preview pending on by default

**Comment by thirstyice** [comment]
- [ ] Make sure that all the relays are parked on. Currently relay 28 isn't (the relay for area 4 + 5 toplight)
